pwndbg/commands/got.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import subprocess
+import struct
+
+import pwndbg.commands
+import pwndbg.commands.checksec
+import pwndbg.enhance
+import pwndbg.chain
+import pwndbg.which
+
+from pwndbg.color import *
+
+
+@pwndbg.commands.Command
+@pwndbg.commands.OnlyWhenRunning
+@pwndbg.commands.OnlyWithFile
+def got():
+    '''
```

Comment:
Not sure how to fix this. Are you suggesting to retrieve the got table from a library routine and then deref the pointer inside the command?